[inductor][test] Fix test_vertical_pointwise_reduction_fusion

henrylhtsang opened the pull request:
Summary: Fix after https://github.com/pytorch/pytorch/pull/131649 changes behavior for fusion.

Test Plan: ci

Differential Revision: D61165949


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

ColinPeppler reviewed (approved):
reminder to re-run with lintrunner!

henrylhtsang committed:
[inductor][test] Fix test_vertical_pointwise_reduction_fusion (#133276)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/133276

Fix after https://github.com/pytorch/pytorch/pull/131649 changes behavior for fusion.

Test Plan: ci

Reviewed By: ColinPeppler

Differential Revision: D61165949
